### DISCUSSION TITLE
missing bottle for qt (big_sur)

### DISCUSSION THREAD

**Discussion by iltommi** [Homebrew/discussions#4251] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.1-91-g6b7ecd1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6b7ecd18e9b27e0a42a66ba8a955e7b57d06af50
Last commit: 3 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 821ee7cc0c5e33d3826e69ab7e4ef7541eaa3950
Core tap last commit: 4 weeks ago
Core tap branch: master
Core tap JSON: 20 Feb 13:35 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.KfxojsWODF/org.xquartz:0
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: octa-core 64-bit skylake
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.7.4-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: 13.2.1
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Hello,
Today I wanted to upgrade packages and it turns out `qt` was upgradable but it started recompiling all of it!

It's missing only big_sur (yes, I know it's quite old, but it should be still supported)
https://github.com/Homebrew/homebrew-core/commit/babf8d05b209f8bd5ba299bfe508084ef2abd5c4

Sure, I can wait to be done (crossing fingers it will work) but qt is huge package, and that's why I use brew ;)
Maybe I have to wait for it to be crafted by the brewbot?
Thanks a lot

**Comment by p-linnane** [comment, marked as answer]
Thank you for reporting this. It looks like we tried to build a new bottle for it [the other day](https://github.com/Homebrew/homebrew-core/actions/runs/4215652947) but ran into some networking issues.

I've kicked off [a new run](https://github.com/Homebrew/homebrew-core/actions/runs/4236246524) which should hopefully be done in a few hours. I will monitor it to make sure it finishes correctly.

**Reply by iltommi** [reply]
thanks in the meanwhile I got it working recompiling

**Reply by p-linnane** [reply]
There is some kind of issue with CI pulling the QT source. I am looking into it.

**Reply by khambrecht** [reply]
Looks like qt bottle is still missing

**Reply by p-linnane** [reply]
Thanks for your patience while I worked through this. `qt` has been rebottled: https://github.com/Homebrew/homebrew-core/commit/24c9ee9c5bf520df26e234eafcc633fcd357c7bb
